"""**Техническое задание:**

1. Создайте функцию имя например min_max, которая:
    1. принимает аргумент: список чисел,
    2. возвращает кортеж из двух чисел. Первое - минимальное число в исходном списке, второе - максимальное число в исходном списке,
    3. В случае, если эти два числа равны, функция поднимает исключение ValueError,
    4. Проверьте работоспособность(см ниже примеры) - создайте тесты в этом же файле. Используйте конструкцию __name__.
2. Создайте еще один файл Python.
3. Импортируйте в него файл с созданной функцией как модуль под псевдонимом utils.
4. Проверьте работоспособность функции на аналогичных примерах.

**Примеры/тесты:**
print(<function_name>([1, 2, 3, 4, 5, 6])) -> 1,6
print(<function_name>([1, 2, 1, 2, 2, 1])) -> 1,2
print(<function_name>([1, 1, 1, 1, 1, 1]))-> ValueError"""

import loft_task_3 as utils

print(utils.min_max([1, 2, 3, 4, 5, 6]))
print(utils.min_max([1, 2, 1, 2, 2, 1]))

